.github/workflows/uc-oss.yml

```diff
@@ -0,0 +1,77 @@
+name: uc-oss
+
+on:
+  pull_request: # for testing
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    paths:
+      - .github/workflows/uc-oss.yml
+      - mlflow/protos/**
+      - mlflow/store/**
+  schedule:
```

Comment:
@BenWilson2 @harupy This test looks to take all of 13 minutes to run start to finish.  We should probably move this to running on every PR that touches related files, but for now, to verify that it isn't noisy or breaking, I'm adding this to run once a day.